firefox-ios/firefox-ios-tests/Tests/ClientTests/Coordinators/LaunchView/LaunchScreenViewModelTests.swift

```diff
@@ -135,6 +135,337 @@ final class LaunchScreenViewModelTests: XCTestCase {
         XCTAssertTrue(updatedValue)
     }
 
+    // MARK: - Multiple Launch Types Tests
+
+    func testLaunchType_termsOfServiceAndIntro_sequence() {
+        FxNimbus.shared.features.tosFeature.with(initializer: { _, _ in
+            TosFeature(status: true)
+        })
```

Comment:
what do you think about adding this in its own method and set this to false for the other tests. In that case, we won't need to edit these tests in the future if we change the configurations in our `yaml` file unless we are removing the feature flag.